guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p>The images in this PR are not visible to me in reviewNB. Could you please check if that's the case.</p><blockquote>Figure 1. Swimming Pools detection using RetinaNet</blockquote><p>Figure 1. Swimming pools detection using RetinaNet</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Yes, I can't see the images in reviewNB. Not sure if it's supported here.</p>